styles/styles.css

```diff
@@ -0,0 +1,118 @@
+
+* {
+  font-family: "Open Sans", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+h3 { 
+  display: block;
+  font-size: 1.10em;
+  margin-top: 1em;
+  margin-bottom: 1em;
+  margin-left: 0;
+  margin-right: 0;
+  font-weight: bold;
+}
+
+.header {
+  align-items: center;
+  background-color: #FFF;
+  border-top: 3px solid #F5C85F;
+  display: flex;
+  justify-content: center;
+  padding: 20px 0 40px;
+  width: 100%;
+}
+
+.vertical-separator {
+  background: linear-gradient(#FFF, #F5C85F, #FFF);
+  width: 3px;
+  height: 300px;
+  margin: 0px 30px;
+} 
+
+.gray-divider {
+  background-image: url("../resources/triangle.png");
+  background-repeat: repeat-x;
+  background-position: bottom;
+  width: 100%;
+  height: 10px;
+}
+
+.features {
+  align-items: center;
+  background-color: #EEE; 
+  display: flex;
+  flex-direction: row;
+  flex-flow: row wrap;
+  justify-content: space-around;
+  -webkit-box-shadow: inset 0px -5px 6px 0px rgba(0,0,0,0.14);
+  -moz-box-shadow: inset 0px -5px 6px 0px rgba(0,0,0,0.14);
+  box-shadow: inset 0px -5px 6px 0px rgba(0,0,0,0.14);
```

Comment:
No need for the box shadows

Reply:
should i just leave the "box-shadow" property? or how should i do to get the shadow between the features and subscribe sections? I used this tool that one of my coworkers linked to me https://www.cssmatic.com/box-shadow .

Reply:
OK, that tool is pretty awesome. For this property, just use the box-shadow, leave the prefixes (-webkit, -moz) :smile: